### DISCUSSION TITLE
How do you prioritize Technical Debt? Invitation for an experiment on Technical Debt.

### DISCUSSION THREAD

**Discussion by TheDutchDevil** [prawnpdf/prawn#1313] [Q&A]
We're an international team of academic Software Engineering researchers investigating technical debt and debt annotation practices. Technical debt (also known as tech debt or code debt) describes what happens when development teams take actions to expedite the delivery of a piece of functionality or a project that later needs to be refactored. In other words, it’s the result of prioritizing speedy delivery over perfect code.

We aim to gather information on how you interpret the priority of source code fragments that contain technical debt. By participating in this survey, you'll help us create a better understanding of technical debt annotation and management procedures. Your insights will be used to support the prioritization of maintenance activities in software systems. The survey will only take 10 to 15 minutes; you can find the study here (https://tueapse.qualtrics.com/jfe/form/SV_39T0IWub1qC8cmy).

This survey is a continuation of work that has been previously published (See https://cassee.dev/files/satd-and-polarity.pdf), and your responses will help us better understand technical debt. Thanks!

I appreciate any help you can provide.

**Comment by pointlessone** [comment]
Prawn is a community-led project. There's no pressure for delivery speed whatsoever. The codebase is in the state that is deemed acceptable by the community. Whenever anyone in the community thinks otherwise they volunteer to rectify the situation and they do it solely on their judgement.

With the above I don't think Prawn is a good data point for your research.
